packages/ui/src/Input/index.tsx

```diff
@@ -0,0 +1,30 @@
+import { HTMLAttributes } from "react";
+
+interface IInputProperties extends HTMLAttributes<HTMLInputElement> {
```

Comment:
I have a concern with this, 
for example by providing type attribute we can make this component check box radio box, etc. 
should we allow this, 
I think this component can be good for every text or number inputs but other than that we should not allow for type like checkbox, radio etc